src/coreclr/nativeaot/System.Private.DisabledReflection/src/Internal/Reflection/RuntimeTypeInfo.cs

```diff
@@ -187,7 +187,10 @@ protected override bool IsByRefImpl()
             return RuntimeAugments.IsByRefType(_typeHandle);
         }
 
-        protected override bool IsCOMObjectImpl() => throw new NotSupportedException(SR.Reflection_Disabled);
+        protected override bool IsCOMObjectImpl()
+        {
+            return false;
```

Comment:
Unrelated fix ?

Reply:
Yes, it is for symmetry with the other `IsComObjectImpl` implementation that I have removed `PROJECTN` ifdef from.